include/pelz_request_handler.h

```diff
@@ -12,11 +12,11 @@
 #include "charbuf.h"
 
 //The maxim key length
-#define MAX_KEY_LEN 32
+#define MAX_KEY_LEN 1024
 #define MAX_SOC_DATA_SIZE 1024
 
 typedef enum
-{ REQ_ENC = 1, REQ_DEC = 2 } RequestType;
+  { REQ_UNK = 0, REQ_ENC = 1, REQ_DEC = 2 } RequestType;
```

Comment:
Was REQ_UNK intended to be used in case of an error? I see you default request_type to this in pelz_thread, but I don't see anything using the value afterward.

Reply:
`REQ_UNK` was introduced because in `thread_process` a `RequestType` variable was initialized to 0, which the C compiler was perfectly happy with but which the C++ compiler rejected. By naming the 0th entry in the `enum` both compilers are happy.